api/spec/graphql/queries/languages/get_languages_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe Queries::Languages::GetLanguages, type: :request do
+  before(:each) do
+    @user = Language.create!(code: 'fr', name: 'French')
+  end
+  describe '.resolve' do
+    it 'returns every users' do
```

Comment:
Corriger le "it". Il me semble que ca return les languages et pas les users non ?